weekly_Q/Balanced_Parenthesis.py

```diff
@@ -0,0 +1,43 @@
+#a string of some "(" and  ")" as input,
+#output should be an integer number to represent the longest contiguous
+#balanced parenthesis that you can understand from my test example below
+
+def balanced_count(string):
+    #left_p_count = 0
+    string_list = []
+    max_count = 0
+    count = 0
+
+    for i in range(len(string)):
+        ch = string[i]
+        if ch == "(":
+            #left_p_count += 1
+            string_list.append(i)
+        if ch == ")":
+            # if left_p_count > 0:
+            #     left_p_count -= 1
+            if len(string_list) > 0:
+                string_list.pop()
+                count += 1
+            else:
```

Comment:
So you're resetting count when you find an unpaired ')'. But in the case of test6, we have a '(' that is never paired (the third character). That's why the test returns 10 instead of 8